Makefile

```diff
@@ -25,29 +25,42 @@ ifeq (,$(filter %$(BUILD_META),$(TAG)))
 $(error TAG $(TAG) needs to end with build metadata: $(BUILD_META))
 endif
 
+REPO ?= rancher
+IMAGE = $(REPO)/hardened-coredns:$(TAG)
+BUILD_OPTS = \
+	--platform=$(TARGET_PLATFORMS) \
+	--build-arg PKG=$(PKG) \
+	--build-arg SRC=$(SRC) \
+	--build-arg TAG=$(TAG:$(BUILD_META)=) \
+	--target coredns \
+	--tag "$(IMAGE)"
+
 .PHONY: image-build
 image-build:
 	docker buildx build \
-		--platform=$(ARCH) \
-		--build-arg PKG=$(PKG) \
-		--build-arg SRC=$(SRC) \
-		--build-arg TAG=$(TAG:$(BUILD_META)=) \
-		--build-arg ARCH=$(ARCH) \
-		--target coredns \
-		--tag $(ORG)/hardened-coredns:$(TAG) \
-		--tag $(ORG)/hardened-coredns:$(TAG)-$(ARCH) \
+		$(BUILD_OPTS) \
 		--load \
```

Comment:
I saw that in some repos we are using --pull and --load whereas in this one we only have --load, could that be a problem?

Reply:
This should not cause any problems. The image-build make target is only used in local development, and I left the `--pull` option in place if it was already there.